### PR TITLE
search field changed into while loop

### PR DESCRIPTION
**Pull request by r4vanan**
i don't want to turn back after entering into ani-cli tool so i done this. if you like too then pull it else leave it thanks

### CODE REVIEW

**Comment by port19x** [comment]
It doesn't quite behave as expected but I agree with that UX

File: ani-cli [hunk 1]
```diff
@@ -352,14 +352,20 @@ history)
 *)
 	while [ -z "$query" ]; do
 		if [ "$use_external_menu" = "0" ]; then
+			clear
```
Comment (Derisis13) [inline ani-cli:355]: Why clear here?

File: ani-cli [hunk 2]
```diff
@@ -352,14 +352,20 @@ history)
...
 			printf "Search anime: " && read -r query
 		else
 			query=$(: | external_menu "" "Search anime:")
 		fi
 	done
 	query=$(printf "%s" "$query" | sed "s| |%20|g")
 	anime_list=$(search_anime "$query")
-	[ -z "$anime_list" ] && die "No results found!"
+	while [ -z "$anime_list" ]; do
+		clear
```
Comment (Derisis13) [inline ani-cli:364]: There is an escape sequence that sets the cursor to the previous line and deletes it. Use that, not clear.

File: ani-cli [hunk 3]
```diff
@@ -352,14 +352,20 @@ history)
...
+		printf "Search anime: " && read -r query
+		search_anime "$query"
```
Comment (Derisis13) [inline ani-cli:366]: The regex for replacing spaces is missing from here

**Review by Derisis13 — CHANGES_REQUESTED** [review]
Lots of repeated lines, there should be a way to solve it without that.

**Pull request closed without merging**